### ISSUE TITLE
many nan loss when retraining without pretrained model

### ISSUE THREAD

**Issue by sueyhan** [thunil/TecoGAN#18]
Hi,

First of all thanks for providing training code. I was about to retrain the model with my own dataset, without pretrained model. But starting from beginning of training cycle, some of the loss functions always output nan (like D_layer_0_loss, D_layer_1_loss, D_layer_2_loss, D_layer_3_loss and l2_warp_loss).

Is it normal to be like this? I didn't change any source code.

**Comment by FrankLinxzx** [comment]
> Hi,
> 
> First of all thanks for providing training code. I was about to retrain the model with my own dataset, without pretrained model. But starting from beginning of training cycle, some of the loss functions always output nan (like D_layer_0_loss, D_layer_1_loss, D_layer_2_loss, D_layer_3_loss and l2_warp_loss).
> 
> Is it normal to be like this? I didn't change any source code.

try to fix batch size lower

**Comment by cloudlakecho** [comment]
@ghost What neural network model did you try? Would you share the training parameters?
